src/parser/core/CombatLogParser.js

```diff
@@ -204,6 +205,7 @@ class CombatLogParser {
     galecallersBoon: GalecallersBoon,
     harlansLoadedDice: HarlansLoadedDice,
     lustrousGoldenPlumage: LustrousGoldenPlumage,
+    rotcrustedvoodoodoll: RotcrustedVoodooDoll,
```

Comment:
I could've sworn I commented on this as well, not sure why it didn't appear in the review. Could you please camelCase the name as well? 
```suggestion
    rotcrustedVoodooDoll: RotcrustedVoodooDoll,
```